src/app/dynamic/enterprise/metering/component.ts

```diff
@@ -68,6 +71,13 @@ export class MeteringComponent implements OnInit, OnDestroy {
           this._cdr.detectChanges();
         },
       });
+
+    this._meteringService.oldReports().subscribe(reports => {
```

Comment:
```suggestion
    this._meteringService.oldReports().pipe(takeUntil(this._unsubscribe)).subscribe(reports => {
```

Reply:
This only do one HTTP request under the hood, it does not merge with any timer. But I guess you're right and we still should apply takeUntil.